docs/developer/watcher-local-test.md

```diff
@@ -1,142 +1,167 @@
+    kind: Secret
+    metadata:
+      name: tls-watcher-default-kyma-sample-skr
+      namespace: default
+      labels:
+        "operator.kyma-project.io/managed-by": "lifecycle-manager"
+    data:
+      ca.crt: $(cat ca.pem | base64)
+      tls.crt: $(cat signed-cert.pem | base64)
+      tls.key: $(cat signed-cert-key.pem | base64)
+    EOF
+    ```
 
-### Create sample Kyma CR
-Run the following command to generate and apply sample Kyma CR and its corresponding secret on KCP:
-```shell
-cat << EOF | kubectl apply -f -
----
-apiVersion: v1
-kind: Secret
-metadata:
-  name: kyma-sample
-  namespace: default
-  labels:
-    "operator.kyma-project.io/kyma-name": "kyma-sample"
-    "operator.kyma-project.io/managed-by": "lifecycle-manager"
-data:
-  config: $(echo "${$(k3d kubeconfig get skr-local)/0.0.0.0/host.k3d.internal}" | base64)
----
-apiVersion: operator.kyma-project.io/v1alpha1
-kind: Kyma
-metadata:
-  name: kyma-sample
-  namespace: default
-spec:
-  channel: regular
-  sync:
-    enabled: true
-  modules:
-  - name: kcp-module
-EOF
-```
-### Verify that watcher-webhook is installed on SKR
-By checking the `Kyma CR` events, verify that the `SKRWebhookIsReady` ready condition is set to `True`.
-```yaml
-Status:                                              
-    Conditions:                                        
-        Message:               skrwebhook is synchronized
-        Observed Generation:   1               
-        Reason:                SKRWebhookIsReady
-        Status:                True
-        Type:                  Ready
-```
-### Edit kyma on SKR
-1. Run the following command to switch the context to using the SKR cluster
+### SKR cluster setup
+Create a local kyma-runtime (SKR) cluster:
 ```shell
-kubectl config use-context k3d-skr-local
+k3d cluster create skr-local
 ```
-2. Add the following line to the Kyma CR `spec` to trigger the watcher KCP update
+
+
+### Create Kyma CR and remote secret
+1. Switch the context for using KCP cluster:
+    ```shell
+    kubectl config use-context k3d-kcp-local
+    ```
+2. Generate and apply sample `Kyma CR` and its corresponding secret on KCP:
+    ```shell
+    cat << EOF | kubectl apply -f -
+    ---
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: kyma-sample
+      namespace: default
+      labels:
+        "operator.kyma-project.io/kyma-name": "kyma-sample"
+        "operator.kyma-project.io/managed-by": "lifecycle-manager"
+    data:
+      config: $(echo "${$(k3d kubeconfig get skr-local)/0.0.0.0/host.k3d.internal}" | base64)
+    ---
+    apiVersion: operator.kyma-project.io/v1alpha1
+    kind: Kyma
+    metadata:
+      name: kyma-sample
+      namespace: default
+    spec:
+      channel: regular
+      sync:
+        enabled: true
+      modules:
+      - name: template-operator
+    EOF
+    ```
+
+### Watcher installation verification
+By checking the `Kyma CR` events, verify that the `SKRWebhookIsReady` ready condition is set to `True`
```

Comment:
Kyma is stuck in `Error` state with the following event logs:
```log
Warning  ModuleReconciliationError  3s (x6 over 6s)  lifecycle-manager  could not synchronize remote module catalog: could not synchronize remote module catalog: could not apply catalog templates: could not apply module template diff: namespaces "kcp-system" not found
```

Reply:
for this the control-plane property should be set on the module-template